cpp/src/groupby/hash/groupby.cu

```diff
@@ -105,12 +106,15 @@ flatten_single_pass_aggs(std::vector<aggregation_request> const& requests)
       col_ids.push_back(i);
     };
 
+    auto values_type = request.values.type().id() == type_id::DICTIONARY32
```

Comment:
Is there an `is_dictionary` trait that could be used instead? That insulates against future dictionary types.

Reply:
No. I can add one though.